app/src/main/res/raw/data.json

```diff
@@ -1738,5 +1738,2345 @@
+    {"RU": {
+        "word": "рассвет",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "sunset",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "закат",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "second",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "секунда",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "minute",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "минута",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hour",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "час",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "week",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "неделя",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "month",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "месяц",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "year",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "год",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "decade",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "десять лет",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "century",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "столетие",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "millennium",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "тысячелетие",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "winter",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "зима",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "spring",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "весна",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "autumn",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "лето",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "summer",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "лето",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "autumn",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "осень",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "fall",
+        "topics": ["time"]
+        }},
+    {"RU": {
+        "word": "осень",
+        "topics": ["время"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "holiday",
```

Comment:
if you use american variant there should be `vacation`

Reply:
Done